deno/base/1.37-dev/Dockerfile

```diff
@@ -51,7 +52,5 @@ ENV HOME="/srv/noteable"
 WORKDIR /etc/noteable/project
 EXPOSE 50001-50005
 
-RUN cargo init
```

Comment:
not sure why this was added in the first place; removing does not cause any difference in behavior

Reply:
I cargo culted from the rust image